code/modules/power/lighting.dm

```diff
@@ -257,7 +257,7 @@
 	if(on)
 		var/BR = nightshift_enabled ? nightshift_light_range : brightness_range
 		var/PO = nightshift_enabled ? nightshift_light_power : brightness_power
-		var/CO = nightshift_enabled ? nightshift_light_color : brightness_color
+		var/CO = nightshift_enabled && (brightness_color == "#a0a080" || !brightness_color) ? nightshift_light_color : brightness_color
```

Comment:
Design point:
Can't you mix/blend the brightness_color with the painted color? This way nightshifts will still be darker

Reply:
The paint takes the place in brightness_color. The painted colors are still darker in nightshift mode because brightness_power. All it would really do is change the hue

![image](https://user-images.githubusercontent.com/20497797/77478905-2d247300-6df5-11ea-87a6-4d0a51a877d4.png)

![image](https://user-images.githubusercontent.com/20497797/77478915-30b7fa00-6df5-11ea-91cf-cf17e2382bae.png)